ui/components/MesheryPatterns.js

```diff
@@ -696,7 +697,8 @@ function MesheryPatterns({
     const design = parseDesignFile(pattern_file);
     e.stopPropagation();
     designLifecycleModal.openModal({
-      title: name,
+      title: `Deploy design "${name}`,
```

Comment:
@aabidsofi19 are we missing a double quote here?

Reply:
yeah , fixed

Reply:
Oh, perfect.